packages/forms/resources/views/components/rich-editor.blade.php

```diff
@@ -377,6 +377,11 @@ class="trix-button trix-button--dialog"
                     @endif
                     x-ref="trix"
                     wire:ignore
+                    wire:key="{{ $this->getId() }}.{{ $statePath }}.{{ $field::class }}.{{
+                        substr(md5(serialize([
+                            $isDisabled,
```

Comment:
Can you test the PR please? I don't think this works because this variable is never assigned like it is in the other files, it is still a getter closure.